test/e2e_env/universal/observability/meshtrace.go

```diff
@@ -34,12 +37,20 @@ func PluginTest() {
 	obsDeployment := "obs-meshtrace"
 	var obsClient obs.Observability
 
+	GatewayAddressPort := func(appName string, port int) string {
+		ip := env.Cluster.GetApp(appName).GetIP()
+		return net.JoinHostPort(ip, strconv.Itoa(port))
+	}
+
 	BeforeAll(func() {
 		err := NewClusterSetup().
 			Install(obs.Install(obsDeployment)).
 			Install(MeshUniversal(mesh)).
 			Install(TestServerUniversal("test-server", mesh, WithArgs([]string{"echo", "--instance", "universal1"}))).
 			Install(DemoClientUniversal(AppModeDemoClient, mesh, WithTransparentProxy(true))).
+			Install(GatewayProxyUniversal(mesh, "edge-gateway")).
+			Install(YamlUniversal(gateway.MkGateway("edge-gateway", mesh, false, "example.kuma.io", "test-server", 8080))).
+			Install(gateway.GatewayClientAppUniversal("gateway-client")).
```

Comment:
"gateway-client" is not listed in the assertion ([code](https://github.com/kumahq/kuma/pull/5243/files#diff-0b974c3ee0b3ce1a922dfa36bdb6d3878f7bf7bee510ea253e7d530f95e4d903R93-R97)), do we need it?

Reply:
it's used by the `ProxySimpleRequests` function.